test/performance/odp_l2fwd_perf.c

```diff
@@ -617,7 +635,7 @@ static odp_bool_t setup_pktios(prog_config_t *config)
 		return false;
 	}
 
-	for (uint8_t i = 0U; i < config->num_ifs; ++i) {
+	for (i = 0U; i < num_ifs; ++i) {
```

Comment:
This change seems unnecessary. Moving the declaration of i from here to function scope should not be done as it is arguably better to declare such loop variables in the for statement. Also, I am not sure that the introduction of num_ifs (and wait_sec) local variable is very useful as it does not seem to make the code any clearer.